_translations/index_gal.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: signed
+title: Unha carta aberta en apoio a Richard M. Stallman
+description: Unha carta aberta en apoio a reincorporación de Richard M. Stallman á Free Software Foundation
+image: /assets/social-media-preview_gal_GAL.png
+locale: gal_GAL
```

Comment:
Is gal_GAL a vliad locale name?

Reply:
I think it is gal_ES but I can't confirm that

Reply:
Is `<lang_code>_<country_code>`, since the Galicia is not recognized as a country but as a simple region of Spain (with some degree of autonomy, but is not relevant), also the correct `lang_code` is `gl`, so it should be `gl_ES`.
For example, if I remember correctly, the locale for Catalan as `ca_ES` is widely used.

Source: [localeplanet.com](https://www.localeplanet.com/icu/)